src/utils/schemas.js

```diff
@@ -112,18 +112,23 @@ const hotelSchema = Joi.object().keys({
 });
 
 const updateTripSchema = Joi.object().keys({
-  hotelId: Joi.number().allow(null, ''),
+  hotelId: Joi.number(),
   type: Joi.string().strict().trim().valid(
     'one way',
-    'return'
+    'return',
   )
     .required(),
   leavingFrom: Joi.number().strict(),
   goingTo: Joi.number().strict(),
   travelDate: Joi.date(),
-  returnDate: Joi.date().when('type', { is: 'return', then: Joi.required() }).allow(null, ''),
```

Comment:
Ask @b0nbon1 about the omission of '.allow(null, '')' since I think he needs it for his feature